libvmi/libvmi_extra.h

```diff
@@ -166,7 +166,7 @@ status_t vmi_get_struct_member_offset_from_json(
  *
  * @return VMI_SUCCESS or VMI_FAILURE
  */
-status_t vmi_get_bitfield_offset_and_size_from_json(vmi_instance_t vmi, json_object *json,
+status_t vmi_get_bitfield_info_from_json(vmi_instance_t vmi, json_object *json,
```

Comment:
This is a public header and renaming an existing API function like this is no-go. If you strongly think the new API name should be used please introduce the new one and add a comment to the old one to say it's deprecated and to be removed in a future version. Then in a couple releases we can remove the old API.